xclim/indices/_multivariate.py

```diff
@@ -533,7 +533,7 @@ def heat_wave_total_length(
 
     cond = (tasmin > thresh_tasmin) & (tasmax > thresh_tasmax)
     group = cond.resample(time=freq)
-    total_count = group.apply(lambda d, dim=None: rl.windowed_run_count(d, window, dim=dim), dim="time")
+    total_count = group.apply(lambda d: rl.windowed_run_count(d, window, dim='time'))
```

Comment:
You can avoid the lambda altogether. Apply passes keyword arguments to the function.

Reply:
Whoops commented in wrong spot:
https://github.com/Ouranosinc/xclim/pull/293#discussion_r345916778

Reply:
Ok! Did that. Sorry, I'm still not a complete xarray master.

Reply:
No worries, me neither for that matter... there's a lot of stuff in there,